package.json

```diff
@@ -34,11 +34,12 @@
     "rewrite-paths": "replace 'SF:.*src' 'SF:src' coverage/lcov.info",
     "start": "yarn watch",
     "start-test-server": "http-server",
+    "test": "cross-env NODE_ENV=test webpack --config webpack.test.config.js && yarn mocha-headless-chrome",
```

Comment:
I don't know if I agree with this change. I typically run `yarn test:ci` as the full test suite, to run all unit and acceptance tests, so if you want a `yarn test` command, we should just have `"test": "yarn test:ci"`, but I'm also not sure we need a `yarn test` command, since we can just use `yarn test:ci`. I'm okay with it though, if we make it run all the tests.

Reply:
I feel like it's useful to have a plain `test` command so new developers can instantly see which one is the default -- but I'm totally open to suggestions as to what it should say 😄!  

My original attempt was to just run `mocha-headless-chrome`, but that wasn't rebuilding the code as I made changes -- so that's how I ended up with the longer set of commands. I just updated again to use `yarn test:ci` instead.